loops-and-arrays.js

```diff
@@ -42,3 +54,9 @@ const squares = [30, 2, 8, 24, 11];
 // console.log(numbers) geeft: [30, 2, 8, 24, 11];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(numbers) geeft: [27000, 8, 512, 13824, 1331];
+
+console.log (squares);
+for (let i = 0; i < squares.length; i++) {
```

Comment:
Hint: Voor de vierkant volume te berekenen moet je de lengte, breedte en hoogte vermenigvuldigen met elkaar. Alle waarde is het hetzelfde.